src/com/homework/MyLinkedList.java

```diff
@@ -0,0 +1,118 @@
+package com.homework;
+
+
+public class MyLinkedList<E> implements MyList<E> {
+    private int size;
+    private Node<E> firstNode;
+    private Node<E> lastNode;
+
+    public MyLinkedList() {
+        firstNode = new Node<>(null, null, null);
+        lastNode = firstNode;
+        size = 0;
+    }
+
+    @Override
+    public boolean add(E e) {
+        final Node<E> l = lastNode;
```

Comment:
`l` is not a good name. It looks like 1 or I (big letter i). Think about a better name. 
The name should reflect the meaning of this variable.